src/main/java/roomescape/auth/service/AuthService.java

```diff
@@ -2,34 +2,42 @@
 
 import org.springframework.stereotype.Service;
 
+import roomescape.auth.dto.LoginMember;
 import roomescape.auth.dto.LoginRequest;
 import roomescape.auth.dto.LoginResponse;
 import roomescape.common.exception.EntityNotFoundException;
 import roomescape.common.exception.LoginFailException;
+import roomescape.member.domain.Email;
 import roomescape.member.domain.Member;
-import roomescape.member.repository.MemberDao;
+import roomescape.member.domain.Password;
+import roomescape.member.repository.MemberRepository;
 
 @Service
 public class AuthService {
 
-    private final MemberDao memberDao;
+    private final MemberRepository memberRepository;
     private final JwtTokenHandler jwtTokenHandler;
 
-    public AuthService(final MemberDao memberDao, final JwtTokenHandler jwtTokenHandler) {
-        this.memberDao = memberDao;
+    public AuthService(final MemberRepository memberRepository, final JwtTokenHandler jwtTokenHandler) {
+        this.memberRepository = memberRepository;
         this.jwtTokenHandler = jwtTokenHandler;
     }
 
     public LoginResponse login(final LoginRequest request) {
-        Member member = memberDao.findByEmailAndPassword(request.email(), request.password())
-                .orElseThrow(() -> new LoginFailException("이메일 또는 비밀번호가 잘못 되었습니다."));
+        Member member = memberRepository.findByEmailAndPassword(
+                new Email(request.email()), new Password(request.password())
+        ).orElseThrow(() -> new LoginFailException("이메일 또는 비밀번호가 잘못 되었습니다."));
```

Comment:
`findByEmailAndPassword` 메서드를 오버로딩한 편의성 메서드를 제공해주면 어떨까요? 

```suggestion
        Member member = memberRepository.findByEmailAndPassword(request.email(), request.password())
                .orElseThrow(() -> new LoginFailException("이메일 또는 비밀번호가 잘못 되었습니다."));
```

Reply:
오버로딩해 메서드를 제공한다면 좀 더 편하게 클라이언트가 사용할 수 있다고 생각합니다.

좋은 의견이라 생각해요 로키!! 반영해 보겠습니다 👍